### PR TITLE
Sendgrid attachments support

### PR DESCRIPTION
**Pull request by galezra**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature request: #94 

- **What is the current behavior?** (You can also link to an open issue here)
No support for file attachments.

- **What is the new behavior (if this is a feature change)?**
Sendgrid's provider now supports attachments.

### CODE REVIEW

**Comment by scopsy** [comment]
Amazing! I'll take a closer look tomorrow 🤟

File: providers/sendgrid/src/lib/sendgrid.provider.spec.ts [hunk 1]
```diff
@@ -1,31 +1,33 @@
 import sendgridMail from '@sendgrid/mail';
-
 import { SendgridEmailProvider } from './sendgrid.provider';
 
+const mocKconfig = {
+  apiKey: 'SG.1234',
+  from: 'test@tet.com',
+};
```
Comment (scopsy) [inline providers/sendgrid/src/lib/sendgrid.provider.spec.ts:7]: Nice! Liked the reusable mock config

Comment (galezra) [reply providers/sendgrid/src/lib/sendgrid.provider.spec.ts:7]: I'll fix the typo

Comment (scopsy) [reply providers/sendgrid/src/lib/sendgrid.provider.spec.ts:7]: @galezra Haven't even noticed it before you wrote XD

Comment (galezra) [reply providers/sendgrid/src/lib/sendgrid.provider.spec.ts]: Done :)

**Pull request merged**